tests/e2e-pw/specs/shopper/shopper-subscriptions-purchase-free-trial.spec.ts

```diff
@@ -0,0 +1,171 @@
+
+			// Open the subscription product, and verify that the
+			// 14-day free trial is shown in the product description
+			await goToProductPageBySlug( shopperPage, productSlug );
+			await expect(
+				shopperPage.locator( testSelectors.productSubscriptionDetails )
+			).toHaveText( /\s*\/ month with a 14-day free trial\s*/ );
+
+			// Add it to the cart and verify that the cart page shows the free trial details
+			await shopperPage.locator( '.single_add_to_cart_button' ).click();
+			await goToCart( shopperPage );
+			await expect(
+				shopperPage.locator( testSelectors.cartSubscriptionDetails )
+			).toHaveText( /\s*\/ month with a 14-day free trial\s*/ );
+
+			// Also verify that the first renewal is 14 days from now
+			await expect(
+				shopperPage.locator(
+					testSelectors.cartSubscriptionFirstPaymentDate
+				)
+			).toHaveText( `First renewal: ${ renewalDateFormatted }` );
+
+			// Verify that the order total is $0.00
+			await expect(
+				shopperPage.locator( testSelectors.cartOrderTotal )
+			).toHaveText( `$0.00` );
+
+			// Proceed to the checkout page and verify that the 14-day free trial is shown in the product line item,
+			// and that the first renewal date is 14 days from now.
+			await setupCheckout( shopperPage, customerBilling );
+			await expect(
+				shopperPage.locator( testSelectors.checkoutSubscriptionDetails )
+			).toHaveText( '/ month with a 14-day free trial' );
+			await expect(
+				shopperPage.locator(
+					testSelectors.cartSubscriptionFirstPaymentDate
+				)
+			).toHaveText( `First renewal: ${ renewalDateFormatted }` );
+
+			// Pay using a 3DS card
+			const card = config.cards[ '3dsOTP' ];
+			await fillCardDetails( shopperPage, card );
+			await shopperPage.click( testSelectors.checkoutPlaceOrderButton );
+			await shopperPage.frames()[ 0 ].waitForLoadState( 'load' );
+			await confirmCardAuthentication( shopperPage, true );
+			await shopperPage.waitForLoadState( 'networkidle' );
+			await expect(
+				shopperPage.getByRole( 'heading', {
+					name: 'Order received',
+				} )
+			).toBeVisible();
+
+			// Get the order ID so we can open it in the merchant view
+			const orderIdField = await shopperPage.$(
+				testSelectors.checkoutOrderId
+			);
+			orderId = await orderIdField.evaluate( ( el ) => el.textContent );
+			const subscriptionIdField = await shopperPage.$(
+				testSelectors.checkoutSubscriptionId
+			);
+			subscriptionId = await subscriptionIdField.evaluate( ( el ) =>
+				el.textContent.trim().replace( '#', '' )
+			);
+		} );
+
+		test( 'Merchant should be able to create an order with "Setup Intent"', async ( {
+			browser,
+		} ) => {
+			const { merchantPage } = await getMerchant( browser );
+			await goToOrder( merchantPage, orderId );
+			await expect(
+				merchantPage.locator( testSelectors.wcOrderPaymentId )
+			).toHaveText( /\(seti_.*\)/ );
+
+			await goToSubscriptions( merchantPage );
+			const subscriptionRow = await merchantPage.locator(
```

Comment:
Nitpick: we can remove `await`, it isn't needed here.

```
'await' has no effect on the type of this expression.ts(80007)
```

Reply:
Thanks, fixed it.